### PR TITLE
AZ-408: Basic runtime update check

### PR DESCRIPTION
**Pull request by pmikolajczyk41**
image

### CODE REVIEW

File: .github/workflows/e2e-tests-main-devnet.yml [hunk 1]
```diff
@@ -32,78 +33,22 @@ jobs:
...
+#      - name: Restore Cargo Cache
+#        uses: actions/cache@v2
+#        with:
+#          path: |
+#            ~/.cargo/bin/
+#            ~/.cargo/registry/index/
+#            ~/.cargo/registry/cache/
+#            ~/.cargo/git/db/
+#            target/
+#          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```
Comment (pmikolajczyk41) [inline .github/workflows/e2e-tests-main-devnet.yml:45]: Enabling caching reduces docker image size - the issue has been already raised on Slack

**Review by h4nsu — APPROVED** [review]
LGTM

**Comment by fbielejec** [comment]
> image

DAG

**Review by fbielejec — APPROVED** [review]

**Review by fixxxedpoint — APPROVED** [review]
lgtm

**Pull request merged**